components/feature/accounts/src/main/java/mozilla/components/feature/accounts/FxaSupportFeature.kt

```diff
@@ -0,0 +1,109 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package mozilla.components.feature.accounts
+
+import android.content.Context
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.ProcessLifecycleOwner
+import mozilla.components.concept.push.PushProcessor
+import mozilla.components.concept.sync.AuthType
+import mozilla.components.concept.sync.ConstellationState
+import mozilla.components.concept.sync.DeviceConstellation
+import mozilla.components.concept.sync.DeviceConstellationObserver
+import mozilla.components.concept.sync.OAuthAccount
+import mozilla.components.concept.sync.AccountObserver as SyncAccountObserver
+import mozilla.components.service.fxa.manager.FxaAccountManager
+import mozilla.components.support.base.log.logger.Logger
+
+internal const val PREFERENCE_NAME = "mozac_feature_accounts"
+internal const val LAST_VERIFIED = "last_verified_push_subscription"
+internal const val PERIODIC_INTERVAL_MILLISECONDS = 24 * 60 * 60 * 1000L // 24 hours
+
+class FxaSupportFeature(
```

Comment:
`FxaPushIntegration`?

Reply:
I made this into `FxaPushSupportFeature` since we use the term 'Integration' in apps when we need to integrate two features on that side of the fence.